llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -1670,6 +1670,12 @@ def PseudoJump : Pseudo<(outs GPR:$rd), (ins pseudo_jump_symbol:$target), [],
                         "jump", "$target, $rd">,
                  Sched<[WriteIALU, WriteJalr, ReadJalr]>;
 
+// Pseudo for LI of simm32.
```

Comment:
Can you add something to the comment to indicate this is currently purely for experimentation purposes?